rudder-core/src/main/scala/com/normation/cfclerk/xmlparsers/TechniqueParser.scala

```diff
@@ -246,9 +251,9 @@ class TechniqueParser(
    *
    * if name content start with RUDDER_CONFIGURATION_REPOSITORY, the path must be considered relative
    * to root of configuration repository in place of relative to the technique.
-   *
+   * TODO: pass the AgentType here
    */
-  private[this] def parseResource(techniqueId: TechniqueId, xml: Node, isTemplate:Boolean): (TechniqueResourceId, String) = {
+  private[this] def parseResource(techniqueId: TechniqueId, xml: Node, isTemplate:Boolean, agentType:Option[AgentType]): (TechniqueResourceId, String) = {
```

Comment:
Here, it seems that the problem is that `isTemplate` is not sufficiently precise, but having both that parameter and `agentType` is redondant. Can you change `(isTemplate, agentType)` by `templateForAgent:Option[AgentType]`

So that we encode the whole information in just that parameter?